addons/terraform-elastic/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+name: terraform-elastic
+version: 1.0.0
+description: Kubernetes Terraform Controller Provider for Elastic Cloud
+url: https://github.com/oam-dev/terraform-controller
+
+tags:
+  - Terraform Controller Provider
+  - IaC
+  - Cloud Resources
+
+deployTo:
+  control_plane: true
+  runtime_cluster: false
+
+dependencies:
+  - name: terraform
+
+# -------------------------------------Configuration Metadata for a Terraform Addon-------------------------------------
+# provider short name
+shortCloudName: elastic
+
+# The Cloud name of the provider
+completeCloudName: Elastic Cloud
```

Comment:
Why need these fileds?

Reply:
To generate a Terraform addon https://kubevela.net/docs/platform-engineers/addon/terraform.

Reply:
But these info cannot recognized by  cli or ui?these fields are not exist in code.

Reply:
Correct. It's the metadata for the tool to generate all files for terraform-elastic addon.

Reply:
Can we put cloud addon related fields in a nested struct. Because these if other addons do not have them?

Reply:
@wangyikewxgm Sure. Good suggestion.

@Nicola115 Can you remove lines [TestGenerateTerraformConfigurationWorkload](https://github.com/oam-dev/catalog/pull/287/files/8adea5b2a7493b075d68328f5fba70581beab3e9#diff-086c3c95d30c58ff9c4112a392246740ec74da8a6178f99a6ef039ed591e2605R18-R31) from this PR. And let's find a solution to refactor the generation tool.

Reply:
> @wangyikewxgm Sure. Good suggestion.
> 
> @Nicola115 Can you remove lines [TestGenerateTerraformConfigurationWorkload](https://github.com/oam-dev/catalog/pull/287/files/8adea5b2a7493b075d68328f5fba70581beab3e9#diff-086c3c95d30c58ff9c4112a392246740ec74da8a6178f99a6ef039ed591e2605R18-R31) from this PR. And let's find a solution to refactor the generation tool.

@Nicola115 Have you deleted these lines? I noticed you just deleted some of them.